lib/rules/no-extra-parens.js

```diff
@@ -543,6 +543,13 @@ module.exports = {
             BinaryExpression: checkBinaryLogical,
             CallExpression: checkCallNew,
 
+            ClassBody(node) {
+                node.body
+                    .filter(member => member && member.type === "MethodDefinition" && member.computed &&
```

Comment:
`member => member && member.type === ...`
seems member will not be `null` here. 😄

Reply:
It was really an unnecessary check, it's removed now :)